independent-projects/bootstrap/app-model/src/main/java/io/quarkus/bootstrap/model/AppArtifactKey.java

```diff
@@ -70,7 +70,7 @@ public static AppArtifactKey fromString(String str) {
     protected final String classifier;
     protected final String type;
 
-    protected AppArtifactKey(String[] parts) {
+    public AppArtifactKey(String[] parts) {
```

Comment:
It doesn't look like this change is necessary. There is already `public AppArtifactKey(String groupId, String artifactId, String classifier, String type)`

Reply:
It makes the code a lot simpler. This is the ctor that is used by the existing code that reads from quarkus-extensions.properties, I would need to copy the logic.